libs/cypress/src/lib/commands/get-by-test-id.ts

```diff
@@ -1,11 +1,13 @@
-// eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
Question (non-blocking): Why were these changes needed?

Reply:
So for some reason the Cypress custom commands don't get recognised when the files are in typescript unless the commands are declared globally. 
https://github.com/cypress-io/cypress-documentation/issues/2565#issuecomment-651251129